### PR TITLE
Add mode prompt to hybrid key bindings.

### PR DESCRIPTION
**Pull request by ken4306**

### CODE REVIEW

**Comment by sn0cr** [comment]
Hi @ken4306, could you provide more context what this PR solves? I'm not sure what your change does.

**Comment by ken4306** [comment]
Hi @sn0cr,

This PR adds a mode indicator used in fish_vi_key_bindings, to fish_hybrid_key_bindings.
Screenshot from 2022-05-13 17-49-11

**Comment by sn0cr** [comment]
Thanks!

File: functions/fish_right_prompt.fish [hunk 1]
```diff
@@ -74,7 +74,7 @@ function prompt_vi_mode -d 'vi mode status indicator'
 end
 
 function fish_right_prompt -d 'Prints right prompt'
-  if test "$fish_key_bindings" = "fish_vi_key_bindings"
+  if not test "$fish_key_bindings" = "fish_default_key_bindings"
```
Comment (sn0cr) [inline functions/fish_right_prompt.fish:77]: I think this alters the behaviour due to the `not`, doesn't it? I would expect something like this:
```suggestion
  if test "$fish_key_bindings" = "fish_default_key_bindings"
```
Else we would have the indicator every time, even for people (like me) that do not use the feature.

**Comment by ken4306** [comment]
I think mode indicator is for someone use vi keybindings and hybrid keybindings, because they need to know current mode.
So if key bindings is not default key bindings, show the indicator. If it is default key bindings, do nothing.

**Comment by sn0cr** [comment]
> I think mode indicator is for someone use vi keybindings and hybrid keybindings, because they need to know current mode. So if key bindings is not default key bindings, show the indicator. If it is default key bindings, do nothing.

Aah, thank you. You inverted the logic here, got it :+1:

**Pull request merged**